src/lib/containers/FullWidthAlert.tsx

```diff
@@ -59,7 +59,7 @@ function ButtonFromConfig(props: {
   variant: string
 }) {
   const { config, variant, className } = props
-  if (config) {
+  if (config && ('onClick' in config || 'href' in config)) {
```

Comment:
In SWC (currently staging only), I made the mistake of always creating a configuration object, even if there's no onClick/href action. One way to fix this is to ensure at least one of those is present.